app/Models/BlogModel.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class BlogModel extends Model
```

Comment:
The name is not per Laravel naming conventions. It should be Blog, without Model